packages/dtslint/src/index.ts

```diff
@@ -1,22 +1,31 @@
 #!/usr/bin/env node
 
+import { getTypesVersions } from "@definitelytyped/header-parser";
 import { AllTypeScriptVersion, TypeScriptVersion } from "@definitelytyped/typescript-versions";
-import assert = require("assert");
+import { assertNever, deepEquals, readJson } from "@definitelytyped/utils";
 import fs from "fs";
 import { basename, dirname, join as joinPaths, resolve } from "path";
+import { satisfies } from "semver";
+import { pipeline } from "stream/promises";
+import tar from "tar";
+import { createGunzip } from "zlib";
+import { checkPackageJson, checkTsconfig, runAreTheTypesWrong } from "./checks";
+import { TsVersion, lint } from "./lint";
+import { getCompilerOptions, packageDirectoryNameWithVersionFromPath, packageNameFromPath } from "./util";
+import assert = require("assert");
+import tmp = require("tmp");
 
-import { deepEquals } from "@definitelytyped/utils";
-import { checkPackageJson, checkTsconfig } from "./checks";
-import { lint, TsVersion } from "./lint";
-import { getCompilerOptions, packageNameFromPath } from "./util";
-import { getTypesVersions } from "@definitelytyped/header-parser";
+const tmpDir = tmp.dirSync().name;
```

Comment:
We may want `tmp.setGracefuleCleanup()` if we aren't deleting the files (I may have missed it below).